specification/v1/core/schema/trqp_request_schema.json

```
@@ -0,0 +1,43 @@
+{
+  "$id": "trqp-request",
+  "title": "TRQP Request",
+  "type": "object",
+  "required": [
+    "entity_id",
+    "assertion_id",
+    "authority_id"
+  ],
+  "properties": {
+    "entity_id": {
+        "type": "string",
+        "description": "The Entity being queried about. "
```

let's try not to reuse words
```suggestion
        "description": "The identifier of the subject the assertion is made for. 
```

We really should link it to the terminology docs